src/components/v5/frame/ColonyHome/partials/ReputationChart/ReputationChart.tsx

```diff
@@ -36,7 +37,11 @@ const ReputationChart = () => {
     colony: { domains },
   } = useColonyContext();
 
-  const { contributorsList } = useContributorsByDomain();
+  const { contributorsList, loading: isContributorsLoading } =
+    useContributorsByDomain();
+
+  // @TODO here we should also check if the colony data is loading
+  const isDataLoading = isContributorsLoading;
```

Comment:
Is this TODO still needed?

Reply:
I left this `TODO` only due to the fact the domain data comes from the colony context and by the way the code is implemented right now, we show the global loader until the colony data is fetched, hence it didn't really make sense to add the loading condition also here, at least for the moment